src/packages/cli/src/CLI.ts

```diff
@@ -119,11 +129,16 @@ export class CLI implements Command {
                 init   Setup Prisma for your app
           introspect   Get the datamodel of your database
             generate   Generate artifacts (e.g. Prisma Client)
-              format   Formats your schema
+              studio   Open Prisma Studio
+              format   Format your schema
+                  db   Manage your database schema and lifecycle ${chalk.dim(
+                    '(preview)',
+                  )}
```

Comment:
`db` is not GA, so should only be visible when run with `--preview` (similar to `migrate` and `--experimental`)

Reply:
As discussed, product decided to not hide commands in the Preview state